### PR TITLE
Finished checkout module

### PR DESCRIPTION
**Pull request by WouterDeLaet**

### CODE REVIEW

File: src/Modules/Checkout/CalculateCheckout/CalculateCheckout.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <div data-testid="calculateCheckoutComp">
+      <ul>
+        <li>
+          <span>Total:</span>
+          <span data-testid="total">{calculateTotal()}</span>
+        </li>
+        <li>
+          <span>Shipping cost:</span>
+          <span data-testid="shippingCost">10</span>
+        </li>
```
Comment (SeppeMarien) [inline src/Modules/Checkout/CalculateCheckout/CalculateCheckout.jsx]: Shipping cost should only been shown when the total is less then 40 euros

File: src/Modules/Checkout/CalculateCheckout/CalculateCheckout.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    3: { product: { id: 3, name: 'cola', image: 'zxcv', sku: 'tyui', price: 30 }, amount: 1 },
+  };
+}
+describe('Calculate checkout', () => {
+  test('Loads by default', () => {
+    const { container } = renderCheckoutList();
+
+    expect(container).not.toBeEmpty();
+  });
+
```
Comment (SeppeMarien) [inline src/Modules/Checkout/CalculateCheckout/CalculateCheckout.spec.jsx:37]: Test edge cases:
-> total= 39,99
-> total= 40.00

**Review by SeppeMarien — CHANGES_REQUESTED** [review]
Please fix following issues

**Review by SeppeMarien — APPROVED** [review]
